content/concepts/kata/translations.md

```diff
@@ -5,73 +5,124 @@ sidebar: explanation-kata
+- The translator earns rank progress for the individual language in which the translation was authored in, equivalent to solving the kata in that language.
+- The translator gains twice the [honor](/references/gamification/honor/#other) compared to solving the kata.
+- If the kata is in beta, rank and honor rewards are granted upon approval of the kata itself.
+- The translator is listed as a [coauthor (TODO: explain)]() of the kata.
 
-_*WARNING:* Currently, "Coauthor" flag seems to be affected by a bug which prevents coauthors of a kata from collecting a reward for its translations. Until it gets fixed, users are rewarded for a translation only if they are not a coauthor of the translated kata yet._
+_WARNING: Currently, the co-author flag seems to be affected by a bug which only rewards translators with honor and rank progress if they have not been listed as a co-author of the kata prior to the approval of the translation._
 
-## Pending translations
+## Reviewing and working with pending translations
 
-TBD
+When a translation is published, it's put into a pending state and awaits for review and approval. Sufficiently privileged users can search for them, review them, and request some fixes, approve, or reject them. Currently, pending translations can be found in one of following ways:
 
-- `Translations` tab on a kata
-- `Translations` page available to "mods"
+- Reading through the kata discourse and searching for posts where translators announced the fact that they published a translation. Translators are encouraged to post a message labelled as **Suggestion** in the kata discourse with a link and information about the language of the translation, so others could find it easily.
+- Navigating to `https://www.codewars.com/kata/kata-id/translations` page lists all accepted and pending translations of a kata.
+- A small group of users has been granted the privilege of accessing the page with a list of all pending translations eligible for approval by them. This is the most convenient way of searching for pending translations, but not available to every user at this moment.
 
-## Reviewing a translation
+### Reviewing a translation
 
-TBD
+Before a pending translation is accepted and a new language becomes available for a kata, it needs to be reviewed and determined whether it holds up to Codewars quality standards. When reviewing a translation, users can perform the following steps:
 
-- HOWTO properly review
+- **Reading the code and description**.
+- **Running the tests** against _other_ solutions along with the reference solution. Unfortunately, testing solutions to a proposed translation other than the reference solution is currently rather cumbersome. One possibility is to fork the translation, replace the reference solution with your solution, run the tests and discard the fork afterwards. Another way is to create a new kumite from scratch and copy/paste code snippets from the translation under review to your kumite.
```

Comment:
>  **Running the tests** against _other_ solutions along with the reference solution.

this sentence isn't clear enough: I know the process and I really have a hard time to figure out what you're talking about, here :s

I guess it's about the order in which you present the information, here. I'd split that in 2 different bullet points

1. running the tests and look at the output (would need a screenshot to show the stuff to click)
2. "if you want to test the code against another solution, ..." -> that's the second idea you're conveying in the current §

Reply:
note: there is that possible bug about: "I want to test the translation, so I open a fork of it and play there with the code, it's ok, so I just go out of the fork without sving/publishing it".
And there... you already lost the hability to gain % in that language, just like if you approve the translation, even if you're not the one who approve it.

not sure this is documented. IIRC, Unnamed posted something about that, but not sure it's in the github repo.

Reply:
yup, it's not very clear. I will think about it.
I do not know how the bug you mentioned works though.

Reply:
thing is, since it's a bug, I don't really know either... x)

Maybe Unnamed could bring more efficient feedback about that (I didn't approve translation in months if not more)

Reply:
> note: there is that possible bug about: "I want to test the translation, so I open a fork of it and play there with the code, it's ok, so I just go out of the fork without saving/publishing it".
And there... you already lost the hability to gain % in that language, just like if you approve the translation, even if you're not the one who approve it.

I wasn't aware of this. Is this reported somewhere? It's not obvious how that's possible so I'll need to dig in deeper later.

Reply:
@error256 Does this happen every time? I was only aware that approving makes you a "coauthor" and that prevents gaining progress.

Reply:
(I just hope my memory is correct about this thing...)
(did I already tell you guys I don't trust my memory more than often...? :/ )